releasenotes/notes/53906.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+releaseNotes:
+- |
+  **Added** Support for reconciling in-pod iptables rules of existing ambient pods on `istio` upgrade. Feature can be toggled with `--set cni.ambient.reconcileIptablesOnStartup=false`
+upgradeNotes:
+  - title: Ambient pod upgrade reconcilation.
+    content: |
+      When a new `istio-cni` Daemonset pod starts up, it will inspect pods that were previously enrolled in the ambient mesh, and upgrade their in-pod iptables rules to the current state if there is a delta. This is on by default as of 1.25.0 and 1.24.3. Installing with a compatibility version target of a previous release will turn this off by default. Feature can be disabled by `helm install cni --set ambient.reconcileIptablesOnStartup=false` (helm) or `istioctl install --set values.cni.ambient.reconcileIptablesOnStartup=false` (istioctl)
```

Comment:
> This is on by default as of 1.25.0 and 1.24.3.

Wasn't 1.24.x supposed to have it off by default?

Reply:
Yep, I removed the other references to 1.24.x but forgot this one.